### PR TITLE
feat(ui-cmd): add command details on variants commands panel

### PR DESCRIPTION
**Pull request by hdinia**
ANT-2536

### CODE REVIEW

**Comment by MartinBelthle** [comment]
3 remarks:

1) For some reason, when clicking on the button to see the commands list, the front-end does 4 calls (2 times the same 2 calls) and I think it should only perform the 2 first. For long lists of commands it could be a little time consuming

2) I tested on my local env and even if the back-end sends the info 
`[{"id":"74403a00-e2a8-454a-aec2-e3770ded7866",
"action":"create_area","args": "area_name":"area_1"},"version":1,"user_name":"admin","updated_at":"2024-12-11T16:15:34.281725"}]`, I see Unknown and unknown for user_name and the date :thinking:  

3) What is the 3rd info displayed in the front ? It's written unknown but i don't really know what's supposed to be displayed

**Comment by hdinia** [comment]
> 3 remarks:
> 
>     1. For some reason, when clicking on the button to see the commands list, the front-end does 4 calls (2 times the same 2 calls) and I think it should only perform the 2 first. For long lists of commands it could be a little time consuming
> 
>     2. I tested on my local env and even if the back-end sends the info
>        `[{"id":"74403a00-e2a8-454a-aec2-e3770ded7866", "action":"create_area","args": "area_name":"area_1"},"version":1,"user_name":"admin","updated_at":"2024-12-11T16:15:34.281725"}]`, I see Unknown and unknown for user_name and the date 🤔
> 
>     3. What is the 3rd info displayed in the front ? It's written unknown but i don't really know what's supposed to be displayed

Thanks for the feedback. Let me clarify each point:

- Those duplicate API calls should not be due to my recent changes, it's legacy code that's pretty messy. We should definitely rework this view at some point.

- About the "Unknown" values, it's intentional that we show "Unknown" as a default state when fields are empty. This maintains UI consistency and avoids having empty/blank spaces in the interface. While you're seeing this now, this case should actually become pretty rare over time, new studies will always have a user and timestamps associated with them. You think we should  display nothing instead ?

- The third field is just showing the last update time in a human-readable format. Like "1 hour ago"

**Comment by MartinBelthle** [comment]
Reply to Hatim last comment:

1- Indeed it's legacy i just wanted to point it out as you're modifying the view. But if it's too complex to refactor we'll keep it this way

2- I agree that we're supposed to see "Unknown" if the back returns empty values but for new commands you'll have the info and they should be displayed

3- The third field isn't a duplicate of the 2nd ?

File: webapp/src/components/App/Singlestudy/Commands/Edition/DraggableCommands/CommandListItem/CommandDetails.tsx [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    },
+    {
+      icon: <ScheduleOutlined sx={{ fontSize: 20 }} />,
+      text: argsData.created ? formatDate(argsData.created) : "Unknown",
+      label: t("global.date"),
+    },
+    {
+      icon: <UpdateOutlined sx={{ fontSize: 20 }} />,
+      text: argsData.lastsave
+        ? formatDateWithLocale(argsData.lastsave, i18n.language)
```
Comment (skamril) [inline webapp/src/components/App/Singlestudy/Commands/Edition/DraggableCommands/CommandListItem/CommandDetails.tsx]: Use `getCurrentLanguage` from i18nUtils. Then you can remove the second parameter and call it directly in `formatDateWithLocale`.

From the doc: 

> If you need the primary used language depending on your configuration (supportedLngs, load) you will prefer using i18next.languages[0].

File: webapp/src/components/App/Singlestudy/Commands/Edition/DraggableCommands/CommandListItem/CommandDetails.tsx [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+  });
+};
+
+function CommandDetails({ item }: Props) {
+  const [t, i18n] = useTranslation();
+
+  const args = item.args as CommandArgsDTO;
+  const argsData = args?.data?.antares || {};
+
+  const details = [
```
Comment (skamril) [inline webapp/src/components/App/Singlestudy/Commands/Edition/DraggableCommands/CommandListItem/CommandDetails.tsx:46]: Use `t("global.unknown")` for all labels with `"Unknown"`

**Review by skamril — CHANGES_REQUESTED** [review]

File: webapp/src/components/App/Singlestudy/Commands/Edition/DraggableCommands/CommandListItem/CommandDetails.tsx (hunk 1 above)
Comment (hdinia) [reply webapp/src/components/App/Singlestudy/Commands/Edition/DraggableCommands/CommandListItem/CommandDetails.tsx]: thanks, done

File: webapp/src/components/App/Singlestudy/Commands/Edition/DraggableCommands/CommandListItem/CommandDetails.tsx (hunk 2 above)
Comment (hdinia) [reply webapp/src/components/App/Singlestudy/Commands/Edition/DraggableCommands/CommandListItem/CommandDetails.tsx:46]: done

**Comment by MartinBelthle** [comment]
I tested on my local env and the bullet points 2 and 3 are resolved IMO

**Review by skamril — APPROVED** [review]

**Pull request merged**